javascript/javascript1/week2/studentManager.js

```diff
@@ -0,0 +1,38 @@
+const class07Students = [];
+
+function addStudentToClass(studentName) {
+  // You write code here
+    if (studentName.trim().length !== 0) {
+        if (class07Students.length < 6 || studentName === 'Queen Margrethe II'){
```

Comment:
really impressed the use of nested if/else statement 🥇 

eagered to check your code by removing some entires in the arry . it works perfect ! it does not take double entries of the same name and does not assign two queen 💯 
👍